vtex-apps/google-tag-manager
# Feat/ga4 new refund event

#### What is the purpose of this pull request?

This PR intends to add the `refund` GA4 event to be sent when `vtex:refund` is received.

[`refund` event](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?hl=en&client_type=gtag#refund)|
-|
<img width="891" alt="Captura de Tela 2023-03-13 às 18 52 47" src="https://user-images.githubusercontent.com/36740164/224840511-f3b323d2-286d-46c5-a693-77138a428182.png">




#### How should this be manually tested?

- In your local environment, inside the `google-tag-manager` app repository, go to the branch `feat/ga4-new-refund-event`;
- Login in your preferred account and workspace;
- Link the app;
- Go to the Admin and access the App Store. Look for the Google Tag Manager app and access the `Settings`;
- Check the `Merge Universal Analytics and Google Analytics 4 Events` option;
- Access the store and perform add a refund a order;
- Check the `dataLayer` object (just type `dataLayer` in the console and press enter). The event data should be in the list.

#### Types of changes
